process_config.yaml

```diff
@@ -31,5 +31,3 @@ gases:
 
 mof_array:
   - IRMOF-1
-  - HKUST-1
-  - NU-125 
```

Comment:
These kinds of commits are a sign that this file should be set up outside the repository. Typically the pattern is:
- have an example file named process_config.sample.yample that contains sample values and a note at the top to create your own local version at process_config.yaml. See https://github.com/WilmerLab/HTSOHM-dev/blob/master/settings/sjs.sample.yaml for an example.
- put process_config.yaml in your .gitignore
- then when you use the application, you can copy the file to process_config.yaml, set it up and change it however you want for testing. You'll probably just leave the simplified version locally for ease of testing.

That way you won't be committing these kinds of changes to the repo. In the future though we will commit FINAL config changes to an experiment repo to track them, but that will be later!